Containers/mastercontainer/start.sh

```diff
@@ -261,36 +261,24 @@ chown www-data:www-data -R /mnt/docker-aio-config/session/
 chown www-data:www-data -R /mnt/docker-aio-config/caddy/
 chown root:root -R /mnt/docker-aio-config/certs/
 
+IPv4_ADDRESS_NC="$(dig nextcloud-aio-nextcloud A +short | grep '^[0-9.]\+$' | sort | head -n1)"
+IPv6_ADDRESS_NC="$(dig nextcloud-aio-nextcloud AAAA +short | grep '^[0-9a-fA-F:]\+$' | sort | head -n1)"
```

Comment:
this only works after the initial startup is complete and the nextcloud container is running and the mastercontainer was restarted at least once...

Reply:
ohhh... that is true...